zips/zip-0246.rst

```diff
@@ -221,8 +219,6 @@ The personalization field of this hash is set to::
 
   "ZTxId6SOutN_Hash" (1 underscore character)
 
-The field encodings are specified in ZIP 230 [#zip-0230-issuance-field-encodings]_.
```

Comment:
Likewise here, undo this removal and instead fix its link to point to the correct ZIP 230 section.